EventKit/eventkit-cloud
# fixes trailing slash, adds settings

## Description of Improvement

Fixes an issue with disabling admin login with redirects. 

## How to test

Set  ENABLE_ADMIN_LOGIN to false and ensure that you are redirected to eventkit login page and not django admin login page.

## Quality Assurance 

The following items have been updated:
 
## Screenshots



:smile:
